x/emissions/keeper/queryserver/query_server_topics.go

```diff
@@ -177,3 +177,34 @@ func (qs queryServer) GetRewardableTopics(
 
 	return &types.QueryRewardableTopicsResponse{RewardableTopicIds: rewardableTopics}, nil
 }
+
+func (qs queryServer) GetActiveTopicsAtBlock(
```

Comment:
so you return a pagination cursor right, but how is the user supposed to query for another page of data? I don't see anything that takes in existing page cursors?